src/navigation/orbitalnavigator.cpp

```diff
@@ -134,24 +134,33 @@ namespace {
     };
 
     constexpr openspace::properties::Property::PropertyInfo FollowAnchorNodeInfo = {
+        "FollowAnchorNodeRotation",
+        "Follow Anchor Node Rotation",
+        "If true, the camera will rotate with the current achor node if within a "
+        "certain distance from it. When this happens, the object will appear fixed in "
+        "relation to the camera. The distance at which the change happens is controlled "
+        "through another property."
```

Comment:
Maybe refer to the other property by name? Then it is easier for the user to search for it if they want to change it too.

Reply:
Yeah, I had the same idea. But the name of the property is super long (`FollowAnchorNodeRotationDistance`), and it will always be located right next to where this documentation is. So wasn't sure if it was necessary to include the full name 🤔